pages/starships/[id].js

```diff
@@ -5,6 +5,7 @@ import styled from "styled-components";
 import Image from "next/image";
 import Flex from "@/components/Layout/Flex";
 import StarshipInfo from "@/components/StarshipInfo";
+import StarshipSellerList from "@/components/StarshipSellersList/StarshipSellerList";
 
 const StyledBackLink = styled(Link)`
```

Comment:
Here is another example of many styled components that could be moved into their own files. See the comment above for that if you like :)